frontend/src/tests/ForestClientCard.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { it, describe, beforeEach, expect, afterEach } from 'vitest';
+import { mount } from '@vue/test-utils';
+import type { VueWrapper } from '@vue/test-utils/dist/vueWrapper';
+import ForestClientCard from '@/components/grantaccess/ForestClientCard.vue';
+import type { FamForestClient } from 'fam-app-acsctl-api';
+
+const testActiveClient: FamForestClient[] = [
+    {
+        client_name: 'TEST',
```

Comment:
Hi Nick, I think some of this testing data is reusable, like the forest client numbers, shall we create a constant file for testing data?